tldw_Server_API/app/core/Jobs/manager.py

```
@@ -1677,8 +1698,28 @@ def acquire_next_job(
+                        # Ordering: default priority ASC (lower first), newest by available/created; for 'chatbooks' prefer DESC
+                        if str(domain) == 'chatbooks':
+                            # For chatbooks: higher numeric priority first; tie-break depends on presence of scheduled rows
+                            try:
+                                _has_sched = False
+                                try:
+                                    _r = conn.execute(
+                                        "SELECT 1 FROM jobs WHERE domain=? AND queue=? AND status='queued' AND (available_at IS NOT NULL AND available_at > DATETIME('now')) LIMIT 1",
+                                        (domain, queue),
+                                    ).fetchone()
+                                    _has_sched = bool(_r)
+                                except Exception:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)